apps/wallet-dashboard/lib/utils/migration/groupStardustObjectsByMigrationStatus.ts

```diff
@@ -76,10 +78,19 @@ export function summarizeMigratableObjectValues({
 
         totalIotaAmount += BigInt(outputObjectFields.balance);
         totalNativeTokens += parseInt(outputObjectFields.native_tokens.fields.size);
-        totalIotaAmount += extractStorageDepositReturnAmount(outputObjectFields, address) || 0n;
+
+        const storageDeposit = extractStorageDepositReturnAmount(outputObjectFields, address);
```

Comment:
`extractStorageDepositReturnAmount` checks that the return address is the same as you, we should create a new function that does the opposite